platform/android/src/ch/opengis/qfield/QFieldCameraPictureActivity.java

```diff
@@ -58,24 +62,35 @@ protected void onCreate(Bundle savedInstanceState) {
     private void callCameraIntent() {
         Log.d(TAG, "callCameraIntent()");
         Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
-        if (takePictureIntent.resolveActivity(getPackageManager()) != null) {
-            File pictureFile = new File(getCacheDir(), pictureTempFileName);
-
-            if (pictureFile != null) {
-
-                Uri photoURI = FileProvider.getUriForFile(
-                    this, "ch.opengis.qfield.fileprovider", pictureFile);
-
-                takePictureIntent.putExtra(MediaStore.EXTRA_OUTPUT, photoURI);
-                startActivityForResult(takePictureIntent, 172);
-            }
+        Context context = getApplication().getApplicationContext();
+        File path = context.getExternalFilesDir(null);
+        File pictureFile =
+            File.createTempFile(pictureTempFileName, ".jpg", path);
```

Comment:
You probably want to use the suffix here

Reply:
Is the ".jpg" not enough?

Reply:
The way we had it before was relying on suffix, which I assumed meant a .png would tell the camera app to save as png. 

If that assumption is wrong, we can get rid of the extra suffix string altogether.

Reply:
`createTempFile` expects the suffix separately, so if that's needed we need to split it off